app/controllers/writable_controller.rb

```diff
@@ -5,6 +5,7 @@ class WritableController < ApplicationController
   def build_template_groups(user=nil)
     return unless logged_in?
     user ||= current_user
+    user = current_user if user.id != current_user.try(:id) && !current_user.mod? && !current_user.admin?
```

Comment:
You can't get here if you're not logged in, so you don't need the try

Reply:
Oh tru